### DISCUSSION TITLE
No html tags 'table' in the result of README example execution.

### DISCUSSION THREAD

**Discussion by vb64** [executablebooks/markdown-it-py#334] [Q&A] [answered]
Hi All!

```
pip install markdown-it-py[plugins]==3.0.0
```

Then executed code from markdown-it-py README file with added last line (print function call).

```python
from markdown_it import MarkdownIt
from mdit_py_plugins.front_matter import front_matter_plugin
from mdit_py_plugins.footnote import footnote_plugin

md = (
  MarkdownIt('commonmark', {'breaks': True, 'html': True})
  .use(front_matter_plugin)
  .use(footnote_plugin)
  .enable('table')
)

text = ("""
---
a: 1
---

a | b
- | -
1 | 2

A footnote [^1]

[^1]: some details
""")

html_text = md.render(text)
print("\n")
print(html_text)
```

And I don't see any table related html tag into output.

![screen](https://github.com/executablebooks/markdown-it-py/assets/8738387/a8497d9d-9966-44af-84bb-3ef4e332e23a)

Tried with Python 3.8 and 3.12.

Something went wrong?

**Comment by chrisjsewell** [comment, marked as answer]
Heya, yeh that's not a valid table; you need to have more - or start with a |

You can see if you just try in here:

a | b
- | -
1 | 2

a | b
-- | -
1 | 2

**Comment by vb64** [comment]
Thanks, it worked.

I used the code from the README example. It probably needs to be fixed.
